packages/nuxt/src/module.ts

```diff
@@ -37,6 +42,7 @@ export interface ModuleOptions {
   autoImports?: Array<string | [string, string]>
 }
 
+const resolver = createResolver(import.meta.url)
```

Comment:
FYI: I move the resolver statement to be in the outermost scope. Will this be ok?

Reply:
yes, it should be fine 👍